From 83899b26ad9c0c565cd41ef296e758675dda003d Mon Sep 17 00:00:00 2001
From: Dmitry Rozhkov <dmitry.rozhkov@linux.intel.com>
Date: Thu, 2 Feb 2017 10:37:52 +0200
Subject: [PATCH] Fix return value to be of expected type

The expected type is a pointer, but the code returns a boolean
value. This makes compilation fail with recent gcc.

Upstream-Status: Submitted [https://sourceforge.net/p/collada-dom/patches/10/]

Signed-off-by: Dmitry Rozhkov <dmitry.rozhkov@linux.intel.com>
---
 dom/src/dae/daeMetaGroup.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dom/src/dae/daeMetaGroup.cpp b/dom/src/dae/daeMetaGroup.cpp
index 676141f..d51dd78 100644
--- a/dom/src/dae/daeMetaGroup.cpp
+++ b/dom/src/dae/daeMetaGroup.cpp
@@ -26,7 +26,7 @@ daeElement *daeMetaGroup::placeElement( daeElement *parent, daeElement *child, d
 	(void)offset;
 	daeString nm = child->getElementName();
 	if ( findChild( nm ) == NULL ) {
-		return false;
+		return NULL;
 	}
 	daeElementRef el;
 
-- 
2.7.4

